packages/tutorial/src/part2.tsx

```diff
@@ -0,0 +1,20 @@
+import * as LLMx from 'ai-jsx';
+import { Inline } from 'ai-jsx/core/inline';
+import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+const app = (
+  <Inline>
+    <ChatCompletion>
+      <UserMessage>Come up with the name of a mythical forest animal.</UserMessage>
+    </ChatCompletion>
+    {'\n\n'}
+    {(conversation) => (
+      <ChatCompletion>
+        <UserMessage>Now write a poem about this animal: {conversation}</UserMessage>
+      </ChatCompletion>
+    )}
+  </Inline>
+);
```

Comment:
Hmm, I'm not sure this is the best example of why someone would want to use `Inline`. If I were writing this, I'd just do:

```suggestion
const animalName = memo(
    <ChatCompletion>
      <UserMessage>Come up with the name of a mythical forest animal.</UserMessage>
    </ChatCompletion>
);
const app = (<>
    {animalName}
    {'\n\n'}
    <ChatCompletion>
      <UserMessage>Write a poem about this animal: {animalName}</UserMessage>
   </ChatCompletion>
  </Inline>
</>);
```

Reply:
I think a better `Inline` example would look something like the examples we have in packages/examples/src/inline-chat.tsx or packages/examples/src/inline-completion.tsx.

Reply:
I modified the example to make it more clear why passing context along would be helpful.